### ISSUE TITLE
Failed to connect #2

### ISSUE THREAD

**Issue by scr44tch** [EntySec/ghost#204]
sry for making a new issue but im a bit confused,

i managed to get access to one of my phones and my shield tv, 
but only after i connected my phone via usb and authorized the connection.

on my authorized desktop:
ghost> set rhost 192.168.1.12
[i] RHOST ==> 192.168.1.12
ghost> run
[*] Connecting to 192.168.1.12:5555...
ghost(192.168.1.12:5555)> 

on my pi (not authorized):
ghost> set rhost 192.168.1.12
[i] RHOST ==> 192.168.1.12
ghost> run
[*] Connecting to 192.168.1.12:5555...
[-] Failed to connect to 192.168.1.12:5555!

my pi is on the same switch, same network as my desktop

isnt the ghost framework supposed to get me access to the device without the authorization?
or did i get that wrong?

_Originally posted by @scr44tch in https://github.com/EntySec/ghost/issues/202#issuecomment-731757611_

**Comment by enty8080** [comment]
@scr44tch Try execute `adb connect 192.168.1.12` on PI.

**Comment by sim0nov** [comment]
@scr44tch So, yeah, this is incorrect because **adb has not authorization**. I think problem in your PI, maybe network disconnected.

**Comment by UjjwalDeep** [comment]
> @scr44tch Try execute `adb connect 192.168.1.12` on PI.
> 
> Maybe your PI have no access to the network, because **adb has no authorization**.

It says unrecognised command!
